### ISSUE TITLE
In Simlife the MDI childs still have default icons

### ISSUE THREAD

**Issue by Anixx** [otya128/winevdm#236]
![Clipboard_20190328](https://user-images.githubusercontent.com/1271736/55159048-f461a780-5168-11e9-850c-a81fcab592d6.png)

**Comment by Anixx** [comment]
I also suggest for the case where the resource table has no icon at all, to use the Win3.1-like [-] icon which is already included in Winevdm

**Comment by cracyc** [comment]
They are built in mdi windows but custom one and they explicitly use the default system menu icon.  Even if they were real mdiclient windows they'd still use that icon because the class is created by user which sets the icon (it can be overridden but not in registerclass).

**Comment by Anixx** [comment]
In that case I suggest the fallback menu icon should be obm_close.bmp rather than the "application" icon from the host system.

**Comment by cracyc** [comment]
I disagree.  The over all look is a win95/xp-ntvdm one, that icon would be out of place.  The exception is those which directly use obm_old_close of which Excel 2 and Winword 1 are the only ones AFAIK.

**Comment by Anixx** [comment]
LodeRunner game also uses something like this and it looks very well. The point here is the look of that button is 3D unlike on Win3.1

**Comment by Anixx** [comment]
May I suggest this icon for the purpose of fallback?
![изображение](https://user-images.githubusercontent.com/1271736/98490514-600f6000-223a-11eb-8282-48a2e546f1d7.png)
(pifmgr.dll, 22)

![изображение](https://user-images.githubusercontent.com/1271736/98491243-e331b580-223c-11eb-91ac-684138f940c9.png)
(appwiz.cpl, 251)

Especially the later one as it has all sizes and colors.

**Comment by Anixx** [comment]
Still default
